test/Microsoft.Azure.WebJobs.Host.FunctionalTests/WebJobs.Host.FunctionalTests.csproj

```diff
@@ -168,7 +169,6 @@
     </Reference>
     <Reference Include="Microsoft.WindowsAzure.Storage, Version=7.2.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
       <HintPath>..\..\packages\WindowsAzure.Storage.7.2.1\lib\net40\Microsoft.WindowsAzure.Storage.dll</HintPath>
-      <Private>True</Private>
```

Comment:
I might be missing a bit of context, but since this fix is supposed to provide WindowsAzure.Storage 8.x compatibility, is it a good idea to test against WindowsAzure.Storage 7.2.1.0 with CopyLocal = true?